src/controllers/dynakube/activegate/internal/authtoken/authtoken_reconciler.go

```diff
@@ -1,4 +1,4 @@
-package secrets
```

Comment:
nitpick: to be consistent the file could be called `reconciler.go` as `authtoken/authtoken_reconciler.go` is a bit redundent